### PR TITLE
I6945 test guides

### LINKED ISSUE #6945: Synchronize TestGuides with the Guides component
`TestGuides` is out of sync with the `Guides` component. This will make it hard to maintain and might decrease effectiveness of the tests. Here are some guidelines to help you clean it up:

* Make sure no test passes a state-mapped prop other than `i18n` and `store` (which we inject for convenience).
* Make sure tests are only passing in valid props
* Any time a test relies on a prop value, pass that value explicitly rather than rely on a default from the `getProps()` helper
* Check for and remove redundant tests (those covering the exact same logic)

### PR DESCRIPTION
**Pull request by rebmullin**
I think this was mostly fixed in #7022. This is just small nits..

fixes #6945

### CODE REVIEW

File: tests/unit/amo/pages/TestGuides.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ describe(__filename, () => {
     );
   };
 
-  it('fetches the add-ons for a guide page', () => {
+  it('fetches the addons for a guide page', () => {
```
Comment (willdurand) [inline tests/unit/amo/pages/TestGuides.js]: I believe we write `add-ons` in writing, and `addon` is code because we can't use an hyphen in variable names.

Comment (rebmullin) [reply tests/unit/amo/pages/TestGuides.js]: I think I have seen it both ways in tests. I can change this back and change the other tests here tho.

**Comment by kumar303** [comment]
I can take a quick look at this as well

File: tests/unit/amo/pages/TestGuides.js (hunk 1 above)
Comment (kumar303) [reply tests/unit/amo/pages/TestGuides.js]: All editorial content must use the hyphen (*add-ons*) but that's the only hard rule we live by. Anything goes in the code 😁

**Comment by willdurand** [comment]
> Please do. I r+ed based on the diff, but re-reading the issue, I believe one test case should be updated:
> 
> ```
> it('renders a Guides Page', () => {
> ```

I don't remember why this one hasn't been changed in #7022..

**Comment by kumar303** [comment]
The *renders a Guides Page* test looked a little long to me but I suppose it's fine. What did you want to see changed?

**Comment by willdurand** [comment]
> The _renders a Guides Page_ test looked a little long to me but I suppose it's fine. What did you want to see changed?

Ha! Now I remember 😓 We inject `content` but this is fine because we have no other option, no?

**Comment by kumar303** [comment]
Ah, good catch. The test should not be injecting `content` because that's not even a valid prop, as far as I can tell.

**Comment by rebmullin** [comment]
> can you update the renders a Guides Page test case?

yes, will do. thanks guys : )

**Pull request merged**